int8 bumper1
int8 bumper2
int8 bumper3
int8 bumper4
int8 bumper5
int8 bumper6
int8 bumper7
int8 bumper8

